uber/config.py

```diff
@@ -133,10 +144,6 @@ def BADGE_PRICE(self):
         return self.get_attendee_price(sa.localized_now())
 
     @property
-    def SUPPORTER_BADGE_PRICE(self):
-        return self.BADGE_PRICE + self.SUPPORTER_LEVEL
-
-    @property
```

Comment:
This property hasn't been used anywhere in the codebase or any plugin for quite some time, so I removed it.